server/app/views/dev/DebugContent.java

```diff
@@ -0,0 +1,62 @@
+package views.dev;
+
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.p;
+
+import auth.FakeAdminClient;
+import j2html.tags.specialized.DivTag;
+import views.BaseHtmlView;
+
+public class DebugContent extends BaseHtmlView {
+  public static DivTag devTools() {
+    return div()
+        .withClasses("flex", "flex-col")
+        .with(
+            p("DEVELOPMENT MODE TOOLS:").withClasses("text-2xl"),
+            redirectButton(
+                "admin",
+                "CiviForm Admin",
+                controllers.routes.CallbackController.fakeAdmin(
+                        FakeAdminClient.CLIENT_NAME, FakeAdminClient.GLOBAL_ADMIN)
+                    .url()),
+            redirectButton(
+                "program-admin",
+                "Program Admin",
+                controllers.routes.CallbackController.fakeAdmin(
+                        FakeAdminClient.CLIENT_NAME, FakeAdminClient.PROGRAM_ADMIN)
+                    .url()),
+            redirectButton(
+                "dual-admin",
+                "Program and Civiform Admin",
+                controllers.routes.CallbackController.fakeAdmin(
+                        FakeAdminClient.CLIENT_NAME, FakeAdminClient.DUAL_ADMIN)
+                    .url()),
+            redirectButton(
+                "trusted-intermediary",
+                "Trusted Intermediary",
+                controllers.routes.CallbackController.fakeAdmin(
+                        FakeAdminClient.CLIENT_NAME, FakeAdminClient.TRUSTED_INTERMEDIARY)
+                    .url()),
+            redirectButton(
+                "feature-flags",
+                "Feature Flags",
+                controllers.dev.routes.FeatureFlagOverrideController.index().url()),
+            redirectButton(
+                "database-seed",
+                "Seed Database",
+                controllers.dev.routes.DatabaseSeedController.index().url()));
+  }
+
+  public static DivTag civiformVersionDiv(String civiformVersion, String civiformImageTag) {
```

Comment:
**suggestion (blocking):** make this class initializable and inject its dependencies rather than forcing the caller to inject and then pass them

Reply:
Done.